content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+
+### Consecutive Success Number
+
+The number of times a transaction or its decendant received an approval from majority (quorum) of queried nodes.
+TODO: add more thorough description
+
+### Progeny
+
+TODO: add
+
+### Slush
+
+TODO: add
+
+## Other
+
+### Graph
+
+TODO: introduce the idea in one sentence without technicals
+
+A graph is pair of sets G = (V, E).
+V is a set of elements called vertices (singular: vertex) and E is a set of paired vertices and its elements are called edges.
+An edge {x, y} contains vertices x and y and they are its endpoints.
+A vertex may belong to no edge and in that case it is not joined to any other vertex.
+The set of vertices in a graph is a discrete value, meaning the set of edges is also a discrete value.
+The number of vertices - |V| is the "order" of the graph and the number of edges - |E| is the "size" of the graph.
+
+### Directed Graph
+
+A directed graph comprises of set of vertices connected by "directed" edges - also called arcs.
+A directed graph is represented as a pair of sets G = (V, A), where
+
+* V is a set of elements (vertices, nodes)
+* A is a set of ordered pairs of vertices, called arcs
+
+### Directed Acyclic Graph (DAG)
+
+A [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) contains no directed cycles - each edge is directed from one vertex to another such that following those directions will never form a closed loop.
+A directed graph is a DAG if and only if it can be topolgically ordered by arranging the vertices as a linear ordering that is consistent with all edge directions.
+A toplogical ordering of a directed graph is an ordering of its vertices into a sequence, such that for every edge the start vertex of the edge occurs earlier in the sequence than the ending vertex of the edge.
+A graph that has a topological ordering does not have any cycles, consequently every graph with a topological ordering is acyclic.
+Every acyclic directed graph has at least one topological ordering.
+
+### Distributed System
+
+A system of computers connected by a network - local, hub-based or wide, that work together as one single computer.
+Applications built with distributed systems "apear" to users and other client applications as if they are running on a single machine.
+Distributed systems take advantage of each node's physical and virtual resources (horizontal scalability), minimize overload of the whole system and can have architecture aiming to achieve a certain performance level, speed or other characteristic levels needed for the application running to overcome.
+
+### Partition of a Set
+
+Partiotion of a set is grouping of its elements into non-empty subsets, in such a way that every element is included in exactly one subset.
+
+### Transitive Relation
+
+A set T with elements a, b, c, d -> A{a, b, c, d}
+A relation R on the elements of X that relates
+a to b,
+b to c,
+c to d
+but also relates
+a to c,
+a to d,
+b to d
+is called transitive.
```

Comment:
I would suggest using inline code blocks for each variable/equation in general, as it makes documentation much more readable. It's easy to see what is a work and what is a variable/equation.  For example, in a sentence if you have a variable called a, since _a_ is also a word in English (used many times throughout this comment alone :p) it can get really confusing for readers.

Reply:
Edited it here: https://github.com/optakt/optakt.github.io/pull/13/commits/d9b5fd4dbe3d581029770f925beabfad5d4608bc#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR135
Looks cleared now, WDYT?